src/commands/build.ts

```diff
@@ -0,0 +1,63 @@
+import {Command, flags} from '@oclif/command';
+import chalk from 'chalk';
+import {execSync} from 'child_process';
+import * as path from 'path';
+
+import MANAGED_PATHS from '../common/managed-paths';
+import ServiceConfig from '../common/service-config';
+
+import Install from './install';
+
+const _info = chalk.blue;
+const _error = chalk.red;
+
+export default class Build extends Command {
+  static description = `Create an ${MANAGED_PATHS.ARCHITECT_JSON} file for a service`;
+
+  static flags = {
+    help: flags.help({char: 'h'}),
+    tag: flags.string({
+      char: 't',
+      required: false,
+      description: 'Name and optionally a tag in the ‘name:tag’ format'
+    }),
+    recursive: flags.boolean({
+      char: 'r',
+      default: false,
+      description: 'Whether or not to build images for the cited dependencies'
+    })
+  };
+
+  static args = [];
+
+  async run() {
+    const root_service_path = process.cwd();
+    await this.buildImage(root_service_path);
+    this.exit();
+  }
+
+  async buildImage(service_path: string) {
+    const {flags} = this.parse(Build);
+    const service_config = ServiceConfig.loadFromPath(service_path);
```

Comment:
Should there be a check after this to make sure that a service config was actually found?

Reply:
`loadFromPath` already throws an error and triggers an exit when the path doesn't exist.